src/components/Dashboard.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useEffect } from 'react'
+import { Link } from 'react-router-dom'
+import { Header, Button, Icon, SemanticCOLORS, SemanticICONS } from 'semantic-ui-react'
+import { useUserState } from '../contexts/UserState'
+import constants from '../utils/constants'
+import strings from '../utils/constants'
+import { USER_ROLES } from '../utils/data'
+import { PermissionPolicyType, Filter } from '../utils/generated/graphql'
+import useListApplications from '../utils/hooks/useListApplications'
+import useListTemplates from '../utils/hooks/useListTemplates'
+import usePageTitle from '../utils/hooks/usePageTitle'
+import { TemplateDetails, TemplateInList } from '../utils/types'
+
+const Dashboard: React.FC = () => {
+  const {
+    userState: { templatePermissions, isNonRegistered },
+    logout,
+  } = useUserState()
+  const {
+    templatesData: { templatesByCategory },
+  } = useListTemplates(templatePermissions, false)
+  if (isNonRegistered) {
+    logout()
+    return null
+  }
+
+  usePageTitle(strings.PAGE_TITLE_HOME)
+
+  return (
+    <div id="dashboard">
+      {templatesByCategory.map(
+        ({ templates, templateCategory: { categoryIcon, categoryTitle } }) => (
+          <div key={categoryTitle} className="template-category">
+            <div className="title">
+              {categoryIcon && (
+                <Icon size="large" color="grey" name={categoryIcon as SemanticICONS} />
```

Comment:
Instead of `color="grey"` you can use `className="dark-grey"` to get our custom colour grey.

Either that or map semantic "grey" to `@headersInteractiveLow` in site.variables (line 18)

Reply:
it looks too dark with our custom one, so idk, ?

Reply:
![Screen Shot 2021-05-14 at 1 51 49 PM](https://user-images.githubusercontent.com/5456533/118208825-8c42bc80-b4bb-11eb-9cde-e8ae80043007.png)

I reckon it looks pretty good. Though we probably want to let Deb give some overall design suggestions for this page, so it's fine for now.